test/inbox_SUITE.erl

```diff
@@ -0,0 +1,195 @@
+%%%-------------------------------------------------------------------
+%%% @author nelson vides
+%%% @copyright (C) 2019, Erlang Solutions
+%%% @doc
+%%%
+%%% @end
+%%% Created : 2019-09-05
+%%%-------------------------------------------------------------------
+-module(inbox_SUITE).
+-author('nelson.vides@erlang-solutions.com').
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("exml/include/exml.hrl").
+-include("mongoose_ns.hrl").
+
+%% API
+-export([all/0,
+         groups/0,
+         init_per_suite/1,
+         end_per_suite/1,
+         init_per_group/2,
+         end_per_group/2
+        ]).
+
+%% Test cases
+-export([
+         incoming_any_marker_should_not_affect_inbox/1,
+         outgoing_any_marker_should_not_be_written_to_inbox/1,
+         outgoing_reset_marker_should_reset_unread_counter/1,
+         outgoing_other_marker_should_not_change_unread_counter/1,
```

Comment:
```suggestion
         outgoing_marker_should_not_change_unread_counter/1,
```

Reply:
In this one I want to emphasise the fact that I'm referring to markers other than the one configured as a reset marker, that's why I called it `other marker`. Any suggestion how to emphasise such?

Reply:
Personally, I think it's clear from the context created by the other test which specifically mentions *reset markers*, but that's just my opinion.

Reply:
Perhaps `outgoing_non_reset_marker_should_not_change_unread_counter`?